### ISSUE TITLE
Add-on Request Floccus

### ISSUE THREAD

**Issue by ghost** [fork-maintainers/iceraven-browser#153]
Please add this add-on to default list
https://addons.mozilla.org/en-US/firefox/addon/floccus/

### Who will benefit from it?
Users who use Next cloud for bookmark

**Comment by daffydock** [comment]
As Nextcloud user, I am on board on this one, please.  I hope it may he possible.

**Comment by abhijitvalluri** [comment]
You can add your own add-ons via settings: https://github.com/fork-maintainers/iceraven-browser/pull/50#issuecomment-685187265

You can create your own add-ons list on https://addons.mozilla.org/. 
* Create an account if you do not already have one, and login. Go to "View My Collections" and create a new collection.
* The collection you create will have a URL in this format: `https://addons.mozilla.org/en-GB/firefox/collections/<account_number>/<collection_name>/`
* Note the `account_number` and `collection_name` from above. Enter those values in the Customize settings in Iceraven settings.
* Profit! Now when you go to the Add-ons section in the browser, you will see your custom collection of add-ons. 

So, basically add whatever add-on you want in the collection you create on https://addons.mozilla.org/. You can have access to the Floccus add-on right away!
